src/twisted/web/_newclient.py

```diff
@@ -1498,6 +1498,10 @@ def __init__(self, quiescentCallback=lambda c: None):
         self._quiescentCallback = quiescentCallback
         self._abortDeferreds = []
 
+    def connectionMade(self) -> None:
```

Comment:
I very much think the answer is "no" but since I noticed this I should probably raise it: do we have to care about inheritance here, and calling `super().connectionMade()`? Hopefully this is just private and we totally don't have to care

Reply:
It's a private class, so inheritance isn't an issue. And `LineReceiver` has no connectionMade so I think it's fine.